lib/plugins/aws/deploy/compile/functions/index.js

```diff
@@ -191,17 +189,7 @@ class AwsCompileFunctions {
         newVersionObject);
     }
 
-    // Add function to Outputs section
-    const newOutput = this.cfOutputDescriptionTemplate();
-    newOutput.Value = { 'Fn::GetAtt': [functionLogicalId, 'Arn'] };
-
-    const newOutputObject = {
-      [functionOutputLogicalId]: newOutput,
-    };
-
-    _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Outputs,
-      newOutputObject);
-
+    // Add function versions to Outputs section
     const functionVersionOutputLogicalId = this.provider.naming
```

Comment:
I am assuming this is only added if users opt-in function versioning. Do you think there's a way to compute function version arns on the fly as well?